harness/determined/common/experimental/experiment.py

```diff
@@ -116,7 +116,12 @@ def await_first_trial(self, interval: float = 0.1) -> trial.TrialReference:
         Wait for the first trial to be started for this experiment.
         """
         while True:
-            resp = bindings.get_GetExperimentTrials(self._session, experimentId=self._id)
+            resp = bindings.get_GetExperimentTrials(
+                self._session,
+                experimentId=self._id,
+                orderBy=bindings.v1OrderBy.ASC,
```

Comment:
what's the reasoning for the added order/sort parameters?

Reply:
To ensure that the method's functionality matched its docstring and name (or at least matched my interpretation of the docstring and name).

Reply:
Should I undo?